config/datatypes_conf.xml.sample

```diff
@@ -271,6 +271,104 @@
+    <datatype extension="textsearch" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="vectorstrip" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="wobble" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="wordcount" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <!-- Report formats http://emboss.sourceforge.net/docs/themes/ReportFormats.html -->
+    <datatype extension="dbmotif" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="diffseq" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="excel" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="feattable" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="motif" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="regions" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="seqtable" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="simple" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="table" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="tagseq" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <!-- Sequence formats http://emboss.sourceforge.net/docs/themes/SequenceFormats.html -->
+    <datatype extension="acedb" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="clustal" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="codata" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="embl" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="fitch" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="gcg" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="genbank" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="hennig86" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="ig" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="jackknifer" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="jackknifernon" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="mega" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="meganon" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="ncbi" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="nexus" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="nexusnon" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="phylip" type="galaxy.datatypes.data:Text" subclass="True"/>
+    <datatype extension="phylipnon" type="galaxy.datatypes.data:Text" subclass="True"/>
```

Comment:
Not sure if/how to do this without defining Python classes (which we ought to do anyway in order to add sniffer methods, see discussion about only adding useful datatypes).